OpenRA.Mods.Common/Traits/Cargo.cs

```diff
@@ -301,14 +306,6 @@ void ReleaseLock(Actor self)
 			takeOffAfterLoad = false;
 		}
 
-		public string CursorForOrder(Actor self, Order order)
```

Comment:
How is this related to the adjacent cells?

Reply:
Not at all! It was a early Christmas present!